kratos/utilities/variable_utils.cpp

```diff
@@ -491,4 +456,34 @@ void VariableUtils::UpdateCurrentPosition(
     KRATOS_CATCH("");
 }
 
+void VariableUtils::AuxiliaryInitializeValue(double &rValue)
+{
+    rValue = 0.0;
+}
+
+void VariableUtils::AuxiliaryInitializeValue(array_1d<double, 3> &rValue)
+{
+    rValue = ZeroVector(3);
+}
+
+void VariableUtils::AuxiliaryReduction(
+    const double &rPrivateValue,
+    double &rSumValue)
+{
+#pragma omp atomic
+        rSumValue += rPrivateValue;
+}
+
+void VariableUtils::AuxiliaryReduction(
```

Comment:
i wouldn't call this "Reduction" ... AtomicAdd?

Reply:
Agree. Done.